mlir/test/mlir-vulkan-runner/argmax.mlir

```diff
@@ -0,0 +1,102 @@
+// RUN: mlir-vulkan-runner %s \
+// RUN:  --shared-libs=%vulkan-runtime-wrappers,%mlir_runner_utils \
+// RUN:  --entry-point-result=void | FileCheck %s
+
```

Comment:
Could you add a couple sentences of explanations that cover what algorithm this implements and outline how this implementation works?

Reply:
Updated PR description